### PR TITLE
Gdpr retrieve mam riak

### PR DESCRIPTION
**Pull request by ludwikbukowski**
GDPR retrieval for MAM MUC and MAM PM with riak backend

### CODE REVIEW

**Comment by mongoose-im** [comment]
[6527.1](https://travis-ci.org/esl/MongooseIM/jobs/534828862) / Erlang 19.3 / small_tests / f28a1bba04375f7d5427371b89d5f1e733cb7aea
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2313/6527/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2313/6527/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-05-20_15.15.05/lib.mongooseim.logs/run.2019-05-20_15.15.06/suite.log.html)

**Comment by mongoose-im** [comment]
[6531.1](https://travis-ci.org/esl/MongooseIM/jobs/535041263) / Erlang 19.3 / small_tests / 293949bb82ee800b3e3f4034bbf4f955f5a49be6
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2313/6531/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2313/6531/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-05-20_22.37.17/lib.mongooseim.logs/run.2019-05-20_22.37.18/suite.log.html)

**Comment by mongoose-im** [comment]
[6533.1](https://travis-ci.org/esl/MongooseIM/jobs/535057867) / Erlang 19.3 / small_tests / b6e84431212a5d1e891fe3325c6220fa3d41c274
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2313/6533/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2313/6533/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-05-20_23.21.58/lib.mongooseim.logs/run.2019-05-20_23.21.58/suite.log.html)

File: big_tests/tests/gdpr_SUITE.erl [hunk 1]
```diff
@@ -266,51 +272,88 @@ retrieve_mam_pm(Config) ->
...
+
+        escalus:send(Alice, escalus_stanza:chat_to(Bob, BodyPmAlice)),
+        escalus:send(Bob, escalus_stanza:chat_to(Alice, BodyPmBob)),
+
+
+        mam_helper:wait_for_room_archive_size(Domain, Room, 4),
+
+        BackendModule = choose_mam_backend(Config, mam),
+        maybe_stop_and_unload_module(mod_mam, BackendModule, Config),
+        maybe_stop_and_unload_module(mod_mam_muc, BackendModule, Config),
```
Comment (DenysGonchar) [inline big_tests/tests/gdpr_SUITE.erl]: backends are different for mod_mam and mod_mam_muc

**Pull request merged**